examples/factories/project-factory/main.tf

```diff
@@ -142,7 +165,7 @@ resource "google_compute_subnetwork_iam_binding" "binding" {
   subnetwork = "projects/${local.vpc_host_project}/regions/${split("/", each.key)[0]}/subnetworks/${split("/", each.key)[1]}"
   region     = split("/", each.key)[0]
   role       = "roles/compute.networkUser"
-  members    = concat(each.value, local.network_user_service_accounts)
+  members    = each.value
 }
 
 resource "google_project_iam_member" "host_project_bindings" {
```

Comment:
This resource should now be obsolete too.

Reply:
don't you need to allow users to set network user at host project level from pf?

Reply:
Aren't you doing that on `module.project.shared_vpc_service_config`?

Reply:
This resource applied `local.host_project_bindings` to the host project, but AFAICT that has been moved to the project module.